src/editor/Editor.js

```diff
@@ -435,6 +435,8 @@ define(function (require, exports, module) {
 
         this.on("cursorActivity", function (event, editor) {
             self._handleCursorActivity(event);
+            self.trigger("providerFound");
```

Comment:
This doesn't make sense to me.  Why are you triggering `providerFound` every time the cursor moves?  You would only want to do that if there was a provider found.